extensions/amp-story/1.0/amp-story.js

```diff
@@ -2216,6 +2239,24 @@ export class AmpStory extends AMP.BaseElement {
     if (!this.sidebar_) {
       return;
     }
+    if (isExperimentOn(this.win,'amp-story-branching')) {
+      this.sidebar_.addEventListener('click', e => {
+        if (e.target.tagName === 'A') {
```

Comment:
I wonder: is this necessarily sufficient to know that it was a fragment link?

Reply:
Ahh, you're right. I added in a check to make sure that the link contains "#page=" and I think that should be sufficient to make sure that the following steps don't apply to outlinks. 

I updated the example linked in the PR description to reflect this change.